test/run.d

```diff
@@ -0,0 +1,342 @@
+#!/usr/bin/env rdmd
+/**
+Allows running tests individually
```

Comment:
Probably not an entirely accurate summary anymore.  It may be prudent to elaborate.